src/GraphQL.Tests/Subscription/SubscriptionTests.cs

```diff
@@ -127,13 +129,47 @@ public async Task Subscribe()
         chat.AddMessage(addedMessage);
 
         /* Then */
-        var stream = result.Streams!.Values.First();
+        var stream = result.Streams.ShouldNotBeNull().Values.First();
         var message = await stream.FirstOrDefaultAsync();
 
-        message.ShouldNotBeNull();
-        message.ShouldBeOfType<ExecutionResult>();
-        message.Data.ShouldNotBeNull();
-        message.Data.ShouldNotBeAssignableTo<Task>();
+        message.ShouldBeSimilarTo("""
+            {"data":{"messageAdded":{"from":{"id":"1","displayName":"test"},"content":"test","sentAt":"2024-01-01T00:00:00\u002B00:00"}}}
+            """);
+    }
+
+    [Fact]
+    public async Task SubscribeInt()
```

Comment:
Tests in this case direct usage of SourceStreamResolver, which is what all the field builders use.